routes/api-routes.js

```diff
@@ -59,6 +59,18 @@ module.exports = function(app) {
     }
   });
 
+  app.delete("/api/new_country", function(req, res) {
+    console.log("country:");
+    console.log(req.params.countries)
+    country.destroy({
+      where: {
+        id: req.params.countries
```

Comment:
I'm not sure this delete is going to work since the id and req.params.countries don't go together I don't think. The key "id" will never have a value of a country name string so a delete will never happen. But let's get it up and see how things go. There's not conflicts so it won't affect anything else even if something is off.